packages/devtron-fe-common-lib/package.json

```diff
@@ -0,0 +1,114 @@
+{
+    "name": "@devtron-labs/devtron-fe-common-lib",
+    "version": "0.5.0-patch-1",
+    "description": "Supporting common component library",
+    "type": "module",
+    "main": "dist/index.js",
+    "types": "dist/index.d.ts",
+    "files": [
+        "dist"
+    ],
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/devtron-labs/devtron-fe-common-lib.git"
+    },
+    "author": "Devtron",
+    "license": "ISC",
+    "bugs": {
```

Comment:
- clean up the package json. 
- Check for singleton for react and other stuff. Can those be commoned out?

Reply:
Can cleanup package.json to remove overrides and stuff. Can pick in v2.